sys/include/net/netstats.h

```diff
@@ -41,6 +50,16 @@ typedef struct {
     uint32_t rx_bytes;          /**< received bytes */
 } netstats_t;
 
+/**
+ * @brief   Get a string description for @ref netstats_module_t, for debugging
```

Comment:
Can you fix the indentation of the text after the doxygen command for this and the following two lines?